<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportWebGLWidget</class>
 <widget class="QWidget" name="ExportWebGLWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export WebGL</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Prefix : </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="prefix_lineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Time Steps : </string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="timeSteps_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Initial Time : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="initialTime_lineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Finish Time : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="finishTime_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="export_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Export WebGL Json Files</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
